demisto_sdk/__main__.py

```diff
@@ -323,23 +323,27 @@ def lint(config, dir, **kwargs):
 
 # ====================== format ====================== #
 @main.command(name="format",
-              short_help="Run formatter on a given script/playbook/integration yml file. ")
+              short_help="Run formatter on a given script/playbook/integration/incidentfield/indicatorfield/"
+                         "incidenttype/indicatortype/layout/dashboard file. ")
 @click.help_option(
     '-h', '--help'
 )
 @click.option(
-    "-t", "--file-type", type=click.Choice(["integration", "script", "playbook"]),
-    help="The type of yml file to be formatted.")
+    "-in", "--input", help="The path of the script yml file")
+# -i --in
 @click.option(
-    "-s", "--source-file", help="The path of the script yml file")
+    "-out", "--output", help="The path where the formatted file will be saved to")
+# -o --out
 @click.option(
-    "-o", "--output-file-name", help="The path where the formatted file will be saved to")
+    "-p", "--pack", help="Specify directory of the pack")
```

Comment:
I think we agreed that we will have only `in` and `out` 
The code will understand if it is a pack or file
@bakatzir @orhovy

Reply:
we said input/output(in is a saved word)